### PR TITLE
[Example] Some improvements for example

### PR DESCRIPTION
**Pull request by ibelem**
Some improvements for example js libs, functions and add object detection in model.html

Split irrelevant code for Semantic Segmentation refactor PR in #545.

### CODE REVIEW

**Comment by ibelem** [comment]
@BruceDai please help to review this PR as interns are on vacation and @huningxin may have no time on this.

File: examples/util/base.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ const preferMap = {
   'low': 'LOW_POWER',
 };
 
-const mobilenet_v1_tflite = {
+
+const imageClassificationModels = [{
```
Comment (BruceDai) [inline examples/util/base.js]: @ibelem Such variables **mobilenet_v1_tflite**,  **mobilenet_v2_tflite** were invoked in index.js#L24 of Benchmark Test, this modification would produce ReferenceError of Benchmark Test. Or you could merge this code, I will update Benchmark Test with this modification.

File: examples/util/base.js [hunk 2]
```diff
@@ -207,10 +197,10 @@ const densenet_onnx = {
   },
   intro: 'Dense Convolutional Network (DenseNet) connects each layer to every other layer in a feed-forward fashion. DenseNets have several compelling advantages: they alleviate the vanishing-gradient problem, strengthen feature propagation, encourage feature reuse, and substantially reduce the number of parameters. ',
   paperUrl: 'https://arxiv.org/abs/1608.06993'
-};
+}];
 
-const ssd_mobilenetv1_tflite = {
-  modelName: 'SSD MobileNet v1(TFlite)',
+const objectDetectionModels = [{
+  modelName: 'ssd_mobilenet_v1_tflite',
```
Comment (BruceDai) [inline examples/util/base.js]: @ibelem This modification will let model name of Object Detection example show as 'ssd_mobilenet_v1_tflite' not previous 'SSD Mobilenet v1(TFlite)' on UI.

File: examples/util/base.js (hunk 1 above)
Comment (ibelem) [reply examples/util/base.js]: I prefer to merge this PR at first.

File: examples/util/base.js (hunk 2 above)
Comment (ibelem) [reply examples/util/base.js]: @BruceDai The new UI in #558 will have no such issue. But I am OK to add new fields as below, please confirm if any conflict base on following code.


```
const imageClassificationModels = [{
  modelName: 'Mobilenet v1 (TFLite)',
  modelFormatName: 'mobilenet_v1_tflite',

```

```
const objectDetectionModels = [{
  modelName: 'SSD MobileNet v1 (TFLite)',
  modelFormatName: 'ssd_mobilenet_v1_tflite',
```

I will submit new commit if no issues for you.

Comment (BruceDai) [reply examples/util/base.js]: @ibelem No conflict,  I think it would be OK to do so.

**Comment by ibelem** [comment]
Merge this PR, @BruceDai please update benchmark codes accordingly.

**Pull request merged**